epsilon: 0.7
pole_0: -2.5
kappa: 70
gamma: 40
alpha: 2
lp_gain_in: 0.4
lp_gain_out: 0.6
max_acc_xy: 3
max_acc_z: 3
output_frame_viz: rmf_owl/base_link
ctrl_freq: 100
obs_to: 1  # obstacle timeout [sec]
cmd_to: 1  # command timeout [sec]
